spec/doubleLinkedList_test.js

```diff
@@ -0,0 +1,156 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import DoubleLinkedList from '../src/doubleLinkedList'
+
+chai.use(chaiChange)
+
+describe('Double Linked List', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(DoubleLinkedList).to.be.a('function')
+  })
+
+  describe('insert()', () => {
+    it('Inserts a node (with the provided data) to the tail of the list', () => {
+      const doubleLinkedList = new DoubleLinkedList()
+
+      expect(() => doubleLinkedList.insert('foo'))
+        .to.alter(() => doubleLinkedList.count, { from: 0, to: 1 })
+      expect(doubleLinkedList.getHeadNode().data)
```

Comment:
More on this: let's say I change `Node`'s `data` field to be named `lkhsklkhdsfglkh`.  I would then change `getData` to return that field.  Anything in our codebase that was referencing `node.data` would break. Any where that was referencing `getData` would not.